bonsai.xcodeproj/project.pbxproj

```diff
@@ -74,6 +82,7 @@
 				20EC47A5275D14DC00881E0F /* SwiftGen */,
 				20115259275BCF5E003BEF0B /* Appearance */,
 				20A602FB275BB17000745114 /* Home */,
+				7E3262D1275E23D10045CEA0 /* Data */,
```

Comment:
I think it might be better rename Data folder to Entities

Reply:
This one is in progress. I think we should discuss it together later in context of app architecture

Reply:
fixed